tests/testsuite/build_scripts_multiple.rs

```diff
@@ -639,27 +650,45 @@ fn bar() {
         .file("assets/foo.txt", "foo")
         .file("assets/bar.txt", "bar")
         .build();
-    p.cargo("check").run();
+    p.cargo("check")
+        .masquerade_as_nightly_cargo(&["multiple-build-scripts"])
+        .run();
 
-    // Editing foo.txt won't recompile, leading to unnoticed changes
+    // Checking again without changes won't trigger recompiling
 
-    p.change_file("assets/foo.txt", "foo updated");
     p.cargo("check -v")
+        .masquerade_as_nightly_cargo(&["multiple-build-scripts"])
         .with_stderr_data(str![[r#"
 [FRESH] foo v0.1.0 ([ROOT]/foo)
 [FINISHED] `dev` profile [unoptimized + debuginfo] target(s) in [ELAPSED]s
 
 "#]])
         .run();
 
-    // Editing bar.txt will recompile
+    // Editing foo.txt will also recompile now since they are separate build scripts
+    p.change_file("assets/foo.txt", "foo updated");
```

Comment:
Editing what is being tested should be in a prior commit.  This commit should only be about switching the test to use two build scripts and updating the output